### DISCUSSION TITLE
can't find the docs - only pens

### DISCUSSION THREAD

**Discussion by WebMechanic** [argyleink/open-props#183] [Q&A] [answered]
Hello,

someone mentioned docs for the colour swatches but I couldn't find any docs on the website?
Is it me not finding them or it this (still) to early in the process?

I love the overall concept but need to "sell it" to my team mates; docs could help :D

Cheers!

**Comment by WebMechanic** [comment, marked as answer]
ok, _probably_ found it: **hidden in plain sight** in the repo. 
yet: `yarn dev` in that folder doesn't cut it to see anything useful, the layout's messed up, so there's something else I am doing wrong :) it also appears to be what's already online (hence "docsite") and I've been there to no avail.

help & hints appreciated

**Reply by brittneypostma** [reply]
Unless I'm misunderstand what you are looking for, I believe this is intended to be the docs site: [https://open-props.style/](https://open-props.style/). Is that not what you are looking for?

**Reply by WebMechanic** [reply]
> Is that not what you are looking for?

actually it's not :)
I know that site, but to me it's more of a showcase with a bunch of sample Codepens, and I glanced over them.
That's all well and good, but there is a lack of (written) information about how it all ties together, i.e. none of the (pretty) colour schemes are of any use for me, so I'd need to customise them. But how, properly?

I didn't dug all too deep into the code and repo, just scanned the `props.*.css`. I also don't understand what it's about the .js files with the same names.
I'm intrigued but also a bit puzzled 🧩

I know writing docs is a pita, however, maybe I overlooked sth.

**Reply by argyleink** [reply]
there's lots beyond the codepens, here's an in page link to the colors https://open-props.style/#colors which show various types of usage and a display of each of them. underneath the display of all colors is a guide called Color Theming 101 that breaks down pretty specifically how to use the colors to build stuff (tie it all together type documentation).

there's lots of code examples instead of explanations of code examples. maybe it's too blunt? it's definitely bizarre that it's only values, you're right that they dont really do squat by themselves, and it's confusing how to put them to use. i feel the codepens show you that, but you're right that the codepens aren't the how/why, they're just the what. if i'm understanding you correctly? but to be honest, i'm confused you dont find the open-props website a docsite, it's very much intended to be the reference and not just the marketing page. the repo is intended for custom building and forks, the site is for dev reference and learning. 

have an example of a query you went to the docsite for and couldnt find? customizing color?

**Reply by WebMechanic** [reply]
ok, that's bizarre ... 
I was using a different browser/computer when I posted, and the page content "stopped" before the "Overview" section with those 4 fancy icons. 
The menu links at the bottom lead nowhere. (The menu's a bit "underpresented" me thinks, could need some ♥)

I'm on a different machine now and here I can see **all the things** I was initially looking for! **Excellent!** 👍🏻
So, 'twas some technical hick-up apparently. 😇

Sorry for the confusion. 
I'll check again this evening, if it's indeed a browser problem (extension overload there).

**Comment by WebMechanic** [comment]
I checked again and whatever went wrong, hiding large parts of the site when I was looking at it the first time is no longer present leaving me puzzled :)

Brilliant work Adam!
We are currently working on a new style guide for the relaunch of one of our websites and I will definitely be including Open Props; not as excessively, but intensively!
